src/app/desktop/components/SystemNavbar.js

```diff
@@ -24,38 +27,92 @@ const SystemNavbar = () => {
-      </div>
-      <div onClick={() => ipcRenderer.invoke('min-max-window')}>
-        <FontAwesomeIcon
-          icon={isMaximized ? faWindowRestore : faWindowMaximize}
-        />
-      </div>
-      <div
-        css={`
-          font-size: 18px;
-        `}
-        onClick={() => ipcRenderer.invoke('quit-app')}
-      >
-        <FontAwesomeIcon icon={faTimes} />
-      </div>
-    </Container>
+  const openDevTools = () => {
+    ipcRenderer.invoke('open-devTools');
+  };
+
+  return !osx ? (
+    <MainContainer>
+      <TerminalButton onClick={openDevTools}>
+        <FontAwesomeIcon icon={faTerminal} />
+      </TerminalButton>
+      <Container os={osx}>
+        <div onClick={() => ipcRenderer.invoke('minimize-window')}>
+          <FontAwesomeIcon icon={faWindowMinimize} />
+        </div>
+        <div onClick={() => ipcRenderer.invoke('min-max-window')}>
+          <FontAwesomeIcon
+            icon={isMaximized ? faWindowRestore : faWindowMaximize}
+          />
+        </div>
+        <div
+          css={`
+            font-size: 18px;
+          `}
+          onClick={() => ipcRenderer.invoke('quit-app')}
+        >
+          <FontAwesomeIcon icon={faTimes} />
+        </div>
+      </Container>
+    </MainContainer>
+  ) : (
+    <OsxMainContainer>
+      <Container os={osx}>
+        <div
+          css={`
+            font-size: 18px;
+          `}
+          onClick={() => ipcRenderer.invoke('quit-app')}
+        >
+          <FontAwesomeIcon icon={faTimes} />
+        </div>
+        <div onClick={() => ipcRenderer.invoke('minimize-window')}>
+          <FontAwesomeIcon icon={faWindowMinimize} />
+        </div>
+        <div onClick={() => ipcRenderer.invoke('min-max-window')}>
+          <FontAwesomeIcon
+            icon={isMaximized ? faWindowRestore : faWindowMaximize}
+          />
+        </div>
+      </Container>
+
+      <TerminalButton onClick={openDevTools}>
+        <FontAwesomeIcon icon={faTerminal} />
+      </TerminalButton>
+    </OsxMainContainer>
   );
 };
 
 export default SystemNavbar;
 
-const Container = styled.div`
+const MainContainer = styled.div`
   width: 100%;
   height: ${({ theme }) => theme.sizes.height.systemNavbar}px;
   background: ${({ theme }) => theme.palette.grey[900]};
   -webkit-app-region: drag;
   display: flex;
   align-items: center;
-  justify-content: flex-end;
+  justify-content: space-between;
+  & > * {
+    -webkit-app-region: no-drag;
+    width: 35px;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    transition: background 0.1s ease-in-out;
+  }
+`;
+
+const OsxMainContainer = styled.div`
+  width: 100%;
+  height: ${({ theme }) => theme.sizes.height.systemNavbar}px;
+  background: ${({ theme }) => theme.palette.grey[900]};
+  -webkit-app-region: drag;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
   & > * {
     -webkit-app-region: no-drag;
     width: 35px;
```

Comment:
Too much duplicated code here. Please extract the logic and reduce code complexity